Exercise_2/interactive_commad_list

```diff
@@ -1,7 +1,11 @@
-cd $HOME/HPC_optimization_and_scalability
+cd /scratch/$USER/training-hpc 
+
 cd Exercise_2
+
 cat main_py.bash
+
 cat main_py.bash
```

Comment:
This should probably be `cat calcMatrix.py`

Reply:
Thanks for catching that @joseph-cooke! Done.